plugins/zenoh-backend-example/src/lib.rs

```diff
@@ -0,0 +1,135 @@
+use std::{
+    collections::{hash_map::Entry, HashMap},
+    sync::Arc,
+};
+
+use async_std::sync::RwLock;
+//
```

Comment:
Please restore the right order of header and `use`.

Reply:
fixed